integrations/extensions/starter-kits/language-model-conversational-search/README.md

```diff
@@ -1,8 +1,12 @@
 # Language Model Conversational Search starter kit
 
-You can use this starter kit to configure language models with Watson Assistant for conversational search. This starter kit shows two ways to do this. The first is an example of [how the IBM watsonx tech preview language model and Watson Discovery search can be used to provide a personalized response to a customer](#example-1-connect-your-assistant-to-a-watsonx-language-model-via-custom-extensions). A second example shows how to use [semantic search output as input to OpenAI](#example-2-connect-your-assistant-to-hugging-face-milvus-and-openai-via-custom-extensions).
+This starter kit has multiple examples of how to configure language models with Watson Assistant for conversational search.
 
-The [prerequisites](#prerequisites) are the same for both examples.
+1. The first is an example of [how the IBM watsonx tech preview language model and Watson Discovery search can be used to provide a personalized response to a customer](#example-1-connect-your-assistant-to-a-watsonx-language-model-via-custom-extensions)
+1. The second example shows how to use [semantic search output as input to OpenAI](#example-2-connect-your-assistant-to-hugging-face-milvus-and-openai-via-custom-extensions)
+1. The third example shows how to use [Watson Discovery for search and use those results as input for OpenAI](#example-3-connect-your-assistant-to-watson-discovery-and-openai-via-custom-extensions)
```

Comment:
We should probably tweak these three bullets to use the same structure so it is easier to spot the differences.  For example, right now, the first bullet lists the model and then the search while the second two both list the search and then the model.

Reply:
Bullet points revised to use the same structure for clarity.